docs/how-to-guides/develop-a-tool/customize_an_llm_tool.md

```diff
@@ -58,3 +76,38 @@ Follow the steps to [build and install your tool package](create-and-use-tool-pa
 Here we use an existing flow to demonstrate the experience, open [this flow](https://github.com/microsoft/promptflow/blob/main/examples/tools/use-cases/custom_llm_tool_showcase/flow.dag.yaml) in VS Code extension.  
 - There is a node named "my_custom_llm_tool" with a prompt template file. You can either use an existing file or create a new one as the prompt template file.  
 ![use_my_custom_llm_tool](../../media/how-to-guides/develop-a-tool/use_my_custom_llm_tool.png)
+
+## FAQs
+### Can I customize text box size for my tool inputs?
+Yes, you can add `ui_hints.text_box_size` field in your tool YAML file as below. There are 4 sizes available, arranged from smallest to largest as `xs`, `sm`, `md`, `lg`.
```

Comment:
Add clear size selection guidance. Please add a table and a well-annotated screenshot to show what does the size name mean, and how wide the text box is per different size. This can help user to choose the appropriate size.

size | description | 
xs | extra small|
....

Reply:
Add below table and image to explain the detail about these sizes.